### DISCUSSION TITLE
Thank you! ❤️

### DISCUSSION THREAD

**Discussion by svenjacobs** [samvtran/jetbrains-macos-keybindings-for-all#7] [General]
I just want to say "Thank you!" for these awesome keybindings ❤️ 

I worked with IntelliJ IDEs on macOS for the past decade and now also use them on Linux.
These keybindings are a life saver because I don't have to learn the Linux/Windows shortcuts and don't have to constantly switch between Linux and macOS bindings 😃

**Comment by samvtran** [comment]
Thanks for the kind words! I'm so glad this humble little keymap has been such a useful tool for all the other OS nomads in the world 😄
